cmd/gaiad/cmd/root.go

```diff
@@ -64,7 +70,7 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 				return err
 			}
 
-			return server.InterceptConfigsPreRunHandler(cmd)
+			return server.InterceptConfigsPreRunHandler(cmd,"", appconfig.AppConfig{})  //todo: check if right
```

Comment:
This is correct. Gaia doesn't have a custom config.
```suggestion
			return server.InterceptConfigsPreRunHandler(cmd, "", nil)
```